aci/resource_aci_cloudsubnet.go

```diff
@@ -89,10 +89,16 @@ func resourceAciCloudSubnet() *schema.Resource {
 			},
 
 			"relation_cloud_rs_subnet_to_flow_log": &schema.Schema{
-				Type: schema.TypeString,
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
Is there a reason why this is not computed?

Reply:
I added computed to it